torchgeo/datasets/so2sat.py

```diff
@@ -24,18 +24,24 @@ class So2Sat(NonGeoDataset):
     acquired by the Sentinel-1 and Sentinel-2 remote sensing satellites, and a
     corresponding local climate zones (LCZ) label. The dataset is distributed over
     42 cities across different continents and cultural regions of the world, and comes
-    with a split into fully independent, non-overlapping training, validation,
-    and test sets.
+    with in a variety of different splits.
 
-    This implementation focuses on the *2nd* version of the dataset as described in
-    the author's github repository https://github.com/zhu-xlab/So2Sat-LCZ42 and hosted
-    at https://mediatum.ub.tum.de/1483140. This version is identical to the first
-    version of the dataset but includes the test data. The splits are defined as
-    follows:
+    This implementation covers the *2nd* and *3rd* versions of the dataset as described
+    in the author's github repository https://github.com/zhu-xlab/So2Sat-LCZ42.
 
-    * Training: 42 cities around the world
-    * Validation: western half of 10 other cities covering 10 cultural zones
-    * Testing: eastern half of the 10 other cities
+    The different versions are as follows:
+
+    * Version 2: This version contains imagery from 52 cities and is split into
+        train/val/test as follows:
+            * Training: 42 cities around the world
+            * Validation: western half of 10 other cities covering 10 cultural zones
+            * Testing: eastern half of the 10 other cities
+    * Version 3: A version of the dataset with 3 different train/test splits, as
+        follows:
+            * Random split: every city 80% training / 20% testing (randomly sampled)
+            * Block split: every city is split in a geospatial 80%/20%-manner
+            * Cultural 10: 10 cities from different cultural zones are held back for
+                testing purposes
```

Comment:
Doc formatting here is completely messed up: https://torchgeo--1283.org.readthedocs.build/en/1283/api/datasets.html#torchgeo.datasets.So2Sat

Not sure exactly how to fix this. I would suggest adding empty lines before or after outer bullet points or removing colons from the end of lines or decrease indentation of inner bullet points or make the outer bullet point lines super long (if necessary)